pypi_auth.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+DOMAIN_OWNER="774774188620"
+DOMAIN="linius"
+REGION="ap-southeast-2"
+REPO="pypi-store"
+
+# If you require use of a named profile, run this script like:
+# AWS_PROFILE=prod ./pypi_auth.sh
+
+AUTH_TOKEN=$(
+    aws codeartifact \
+    get-authorization-token \
+    --domain-owner "$DOMAIN_OWNER" \
+    --domain "$DOMAIN" \
+    --profile "$PROFILE" \
+    --region "$REGION" \
+    --query authorizationToken \
+    --output text
+)
+
+export PIP_EXTRA_INDEX_URL="https://aws:$AUTH_TOKEN@$DOMAIN-$DOMAIN_OWNER.d.codeartifact.$REGION.amazonaws.com/pypi/$REPO/simple"
+echo "$PIP_EXTRA_INDEX_URL"
```

Comment:
newline. The export won't work unless you're `source`ing the file btw.

Reply:
I think the export is for if you want to source the file and login/publish manually (we have done this in other packages)

Reply:
Yep just been copying and pasting this helper script around

Reply:
lets just leave it as it is useful